http/bit.lua

```diff
@@ -7,17 +7,26 @@ The bit operations are only done
 This means we can ignore the differences between bit libraries.
 ]]
 
--- Lua 5.3 has built-in bit operators, wrap them in a function.
-if _VERSION == "Lua 5.3" then
-	-- Use debug.getinfo to get correct file+line numbers for loaded snippet
-	local info = debug.getinfo(1, "Sl")
-	return assert(load(("\n"):rep(info.currentline+1)..[[return {
-		band = function(a, b) return a & b end;
-		bor = function(a, b) return a | b end;
-		bxor = function(a, b) return a ~ b end;
-	}]], info.source))()
+-- Lua 5.1's load function doesn't support reading strings, only
```

Comment:
Use `loadstring` in 5.1?

Reply:
Yeah, I guess it's six of one, half-dozen of the other.

This could be replaced with something like "local l = load; if loadstring then l = loadstring"

I can switch to something like that if you prefer, let me know!

Reply:
I guess `loadstring` doesn't have a `source` argument; so `load` is best.

It annoys me a little that we don't use straight up `load` in newer versions of lua.
I just realised we already depend on compat-5.3 in lua 5.1, so how about:
```lua
local load = load
if _VERSION == "Lua 5.1" then
    load = require "compat53.module".load
end
```

Reply:
re: `loadstring` - in the docs they list the second parameter as `chunk` instead of `source`, but it's the same thing, `loadstring` should work and show the source filename as intended.

Reply:
See latest push, I just did `local load = loadstring or load`

Reply:
I don't want to use `loadstring` if `load` is available.